app/SettingsScreen.js

```diff
@@ -0,0 +1,127 @@
+import React, {Component} from 'react';
+import { View, Text, Button, Switch, StyleSheet, Alert, TextInput, AsyncStorage } from 'react-native';
+
+export default class SettingsScreen extends Component{
+	state = {
+		blackBackground: false,
+		username: '',
+	}
+
+	componentDidMount(){
+		this.storageToState();
+	}
+
+	storageToState = async () => { 
+		try {
+  			const value = await AsyncStorage.getItem('@DemoRNStore:settings_background');
+  			if (value !== null){
+    			this.setState({blackBackground: value == 'true'});
```

Comment:
value == 'true', always try to use 3 equation signs, e.g. value === 'true', compiles faster (no need for silent conversions) and gives less unpredictable errors

Reply:
It's say that is unnecessary? Or it's my english is bad?
https://stackoverflow.com/questions/263965/how-can-i-convert-a-string-to-boolean-in-javascript